### PR TITLE
fix: guarantee aws credentials file exists

### PR DESCRIPTION
**Pull request by philipyoo**

### CODE REVIEW

File: tests.bash [hunk 1]
```diff
@@ -2,6 +2,9 @@
 set -x #echo on
 set -e #exit at the first error
 
+mkdir -p ~/.aws
+touch ~/.aws/credentials
```
Comment (jonathan-gramain) [inline tests.bash]: `touch` is not necessary, since `>>` will create the file if does not exist

**Comment by philipyoo** [comment]
@ironman-machine try

**Comment by philipyoo** [comment]
@ironman-machine try

**Comment by rahulreddy** [comment]
Closing as it is no longer relevant.

**Pull request closed without merging**